src/main/resources/avro/variantmethods.avdl

```diff
@@ -51,6 +51,89 @@ SearchVariantSetsResponse searchVariantSets(
   /** This request maps to the body of `POST /variantsets/search` as JSON. */
   SearchVariantSetsRequest request) throws GAException;
 
+
+/******************  /variantsets/{id}/sequences/search  *********************/
+/**
+This request maps to the body of `POST /variantsets/{id}/sequences/search` as
+JSON.
+*/
+record SearchVariantSetSequencesRequest {
```

Comment:
Would `SearchVariantSetSegmentsRequest` mapping to `/variantsets/{id}/segments/search` be more natural here? We're returning lists of `Segments` so it would seem a bit clearer to me.